src/index.coffee

```diff
@@ -10,7 +10,7 @@ class AlphabetizeKeys
     level: 'error'
     message: 'Object and class keys should be alphabetized'
     name: 'alphabetize_keys'
-
```

Comment:
Please add a newline back after the new key. I'm using 2 newlines for separation.